api/node/Cargo.toml

```diff
@@ -23,10 +23,13 @@ napi = { version = "2.12.0", default-features = false, features = ["napi8"] }
 napi-derive = "2.12.2"
 i-slint-compiler = { workspace = true, features = ["default"] }
 i-slint-core = { workspace = true, features = ["default"] }
+i-slint-backend-selector = { workspace = true }
 slint-interpreter = { workspace = true, features = ["default", "display-diagnostics", "internal"] }
+slint = { workspace = true, features = ["default"] }
```

Comment:
Why is that needed?   it seems the item you try to access are in i-slint-core or i-slint-backend-selector

But if you really need that, i don't think you should enable the default features here.

Reply:
Good point, I'll try to eliminate the dependency.